### ISSUE TITLE
executeContract got errors when functionArgument is not set

### ISSUE THREAD

**Issue by scalarindetail** [scalar-labs/scalardl-javascript-sdk-base#9]
Report use upper layer's Web SDK to execute a contract with the pattern of
```
clientService.executeContract(contractId, argument); // without passing functionArgument
```
Then, the Scalar server returns
```
JsonParser#getObject() or JsonParser#getObjectStream() is valid only for START_OBJECT parser state. But current parser state is VALUE_NULL
```
Apprently this message is Java runtime error message.
It is likely scalardl-javascript-sdk-base doesn't handle the undefined functionArgument case.

**Comment by scalarindetail** [comment]
Found the reason.

If we use gRPC JavaScript library to specify `null` to a string field, the library will convert it to `"null"` but not `null`. We have to use `undefined` if we want the field to be `null`.

**Comment by scalarindetail** [comment]
Fixed in #10
